app/src/main/java/org/openobservatory/ooniprobe/activity/MainActivity.java

```diff
@@ -109,8 +109,19 @@ else if (getPreferenceManager().getAppOpenCount() == PreferenceManager.NOTIFICAT
     @Override
     protected void onNewIntent(Intent intent) {
         super.onNewIntent(intent);
-        if (intent.getExtras() != null && intent.getExtras().containsKey(RES_ITEM))
-            bottomNavigation.setSelectedItemId(intent.getIntExtra(RES_ITEM, R.id.dashboard));
+        assert intent != null;
```

Comment:
Why do we want to assert here?

Reply:
Removed, it was part of my testing code when I got a crash, intent can't be null there